README.md

```diff
@@ -6,4 +6,8 @@
 # Retool Terraform Provider
 Official plugin for managing Retool configuration via [Terraform](https://www.terraform.io/).
 
-TODO: add links to Terraform Registry, provider docs and "getting started" section.
+
+Find the documentation and examples on our [Terraform Registry page](https://registry.terraform.io/providers/tryretool/retool/latest/docs).
+
+
+For any inquiries regarding using Retool Terraform provider, please feel free to reach out to us at support@retool.com or search our [Community Forums](https://community.retool.com/) and post your question there.
```

Comment:
Users of this will be on enterprise, so they probably wouldn't frequent the community forum right?

Reply:
yeah, but I guess there might be some share of "when on Business/Team/Free" questions.